encord/user_client.py

```diff
@@ -989,12 +990,17 @@ def find_storage_items(
 
         return StorageFolder._list_items(self._api_client, "storage/search/items", params)
 
+    @deprecated("CURRENT", ".metadata_schema()")
     def get_client_metadata_schema(self) -> Optional[Dict[str, ClientMetadataSchemaTypes]]:
         return get_client_metadata_schema(self._api_client)
 
+    @deprecated("CURRENT", ".metadata_schema()")
     def set_client_metadata_schema_from_dict(self, json_dict: Dict[str, ClientMetadataSchemaTypes]):
         set_client_metadata_schema_from_dict(self._api_client, json_dict)
 
+    def metadata_schema(self) -> MetadataSchema:
```

Comment:
As is, looks like / could be property? Possibly a good move

Reply:
not sure about property - it is a network request